<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MohamedMurtuza" ignored="0" name="Test" tests="4" failures="1" timestamp="2020-04-26T04:55:01 IST" time="0.078" errors="0">
  <testcase name="heyU" time="0.013" classname="c.annabella"/>
  <testcase name="it" time="0.003" classname="c.annabella">
    <skipped/>
  </testcase> <!-- it -->
  <testcase name="it" time="0.001" classname="c.annabella">
    <skipped/>
  </testcase> <!-- it -->
  <testcase name="it" time="0.002" classname="c.annabella">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at c.annabella.it(annabella.java:11)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- it -->
</testsuite> <!-- Test -->
